synapse/tests/test_lib_interval.py

```diff
@@ -24,5 +24,5 @@ def test_ival_overlap(self):
         self.true(s_interval.overlap(ival2, ival3))
 
     def test_ival_parsetime(self):
-        ival = s_interval.parsetime('1970-1980')
+        s_interval.parsetime('1970-1980')
```

Comment:
why is this change here?  it doesn't actually mean anything.  was this a machine-change?

Reply:
This probably came from pyflake detecting that ival wasn't used.

Reply:
Yep.  Deleting the line, it's redundant with the line below.